vyper/types/convert.py

```diff
@@ -346,9 +344,9 @@ def to_bytes32(expr, args, kwargs, context):
     if input_type == 'bytes':
         if _len > 32:
             raise TypeMismatchException((
-                "Unable to convert bytes[{}] to bytes32, max length is too "
+                f"Unable to convert bytes[{len}] to bytes32, max length is too "
```

Comment:
Shouldn't this be `_len`? Is this a bug @jacqueswww?
```suggestion
                f"Unable to convert bytes[{l_en}] to bytes32, max length is too "
```

Reply:
I think you meant:
```suggestion
                f"Unable to convert bytes[{_len}] to bytes32, max length is too "
```

Reply:
Yup, fixed that haha

Reply:
@wschwab do this one